smart_campus/app/templates/app/station_new.html

```diff
@@ -1,118 +1,105 @@
 {% extends "app/base.html" %}
 {% load static %}
 
-{% block title %}Smart Campus{% endblock %}
-
 {% block stylesheet %}
-  {{ block.super }}
-  <link rel="stylesheet" type="text/css" href="{% static "app/css/station_new.css" %}" />
+  {{ block.super }}>
+  <link rel="stylesheet" type="text/css" href="https://cdnjs.cloudflare.com/ajax/libs/trix/0.11.0/trix.css">
+  <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/trix/0.11.0/trix.js"></script>
```

Comment:
Why is script tag added here?

Reply:
according to the [trix documation](https://github.com/basecamp/trix#getting-started), it said that "Include the bundled trix.css and trix.js files in the <head> of your page."

Reply:
But, shouldn't the trix.js be included in the `script` block?

Reply:
Okay, fixed it.